frontend/src/metabase/ui/components/inputs/Select/Select.module.css

```diff
@@ -150,10 +143,15 @@
   line-height: var(--select-item-line-height);
 }
 
+.SelectItems_Options {
+  padding: 0.75rem;
+}
+
 .Dropdown {
   background-color: var(--mb-color-background);
   border-color: var(--mb-color-border);
-  overflow: auto;
+  /* need to override the inline style from Mantine */
+  max-height: rem(500px) !important;
```

Comment:
there's one gotcha here, in that on viewports where the whole 500px doesn't fit on the screen, the dropdown won't get smaller to fit in the viewport. Mantine actually does autosize the select, but that's the style we're overriding here.

There's a nice way around it by using the select's default behavior: an autosizing scrollarea, but that has broader consequences than I'm willing to take on in this PR. The value of optimizing for screen sizes less than 500px high seems limited.

Reply:
ok, this actually ends up being a real problem, and even fails tests, certain options become unclickable